### ISSUE TITLE
数据集训练lable问题

### ISSUE THREAD

**Issue by libaitian-oss** [mkang315/CST-YOLO#5]
你好作者，为什么我在训练时会报错Can not train without labels，显示在数据集目录下的images.cache没有标签数据

**Comment by mkang315** [comment]
This error is because none of your images have labels/annotations. Make sure the data path is correct. To train correctly, the label format for images must be in YOLO format (.txt), otherwise there is nothing for YOLO to learn.

**Comment by libaitian-oss** [comment]
Do you have to write the txt file of this lable manually? I didn't find the lable folder in the file linked to your dataset。

**Comment by libaitian-oss** [comment]
Is the txt text of this lable file something I have to write myself? I didn't find the lable folder inside the file linked to your dataset.

**Comment by mkang315** [comment]
Yes. I suppose you can find a source code on GitHub for converting the original labels to any format if they are not YOLO format.

**Comment by libaitian-oss** [comment]
thank you

**Comment by libaitian-oss** [comment]
Why do I keep getting an error after converting the XML format to the YOLO format, saying that "labels" cannot be found?

**Comment by mkang315** [comment]
You need to specify the correct label paths. See the custom training tutorial to get started, and follow the example:
https://docs.ultralytics.com/yolov5/tutorials/train_custom_data

I suggest you'd better try your dataset using the original YOLOv5/7/8 models and find whether it works with them. Thank you!
